tests/clients/github/test_github.py

```
@@ -0,0 +1,176 @@
+from unittest import TestCase
+import string
+from gevent import sleep
+from os import path, getenv
+
+from jumpscale.loader import j
+from tests.base_tests import BaseTests
+
+
+class GithubClientTest(BaseTests):
+    def setUp(self):
+        super().setUp()
+        self.instance_name = j.data.random_names.random_name()
+
+        self.username = getenv("USERNAME")
+        self.password = getenv("PASSWORD")
+        self.email = getenv("EMAIL")
```

Please raise an error if these environment variables are not provided.